lib/que/adapters/active_record_with_lock.rb

```diff
@@ -63,6 +84,21 @@ def unlock_job(job_id)
           conn.execute("SELECT pg_advisory_unlock(#{job_id})")
         end
       end
+
+      def observe(metric, metric_duration, labels = {})
```

Comment:
```suggestion
      def observe(count_metric: nil, duration_metric: nil, labels: {})
```
use named args with defaults here, as it's not always clear what the order of arguments mean, and why we're passing in nil